vmdb/app/controllers/application_controller/tenancy.rb

```diff
@@ -0,0 +1,19 @@
+module ApplicationController::Tenancy
+  extend ActiveSupport::Concern
+
+  included do
+    set_current_tenant_by_subdomain_or_domain(:tenant)
+  end
+
+  # NOTE: remove when these session vars are removed
+  def set_session_tenant(tenant = current_tenant)
+    session[:customer_name] = tenant.try(:name)
+    session[:vmdb_name]     = tenant.try(:vmdb_name)
+    session[:custom_logo]   = tenant.try(:logo?)
```

Comment:
yes. it is a boolean and an implied hardcoded file name.
In the longer term I was thinking this would probably be the logo filename.

Also, in the short term, we'll just tear the `session[:*]` values out of the views and use the current_tenant.name ...